rkt/rkt.go

```diff
@@ -53,11 +53,11 @@ func init() {
 }
 
 type Command struct {
-	Name        string       // Name of the Command and the string to use to invoke it
-	Summary     string       // One-sentence summary of what the Command does
-	Usage       string       // Usage options/arguments
-	Description string       // Detailed description of command
-	Flags       flag.FlagSet // Set of flags associated with this command
+	Name        string        // Name of the Command and the string to use to invoke it
+	Summary     string        // One-sentence summary of what the Command does
+	Usage       string        // Usage options/arguments
+	Description string        // Detailed description of command
+	Flags       *flag.FlagSet // Set of flags associated with this command
```

Comment:
The circular initialization upsets Go, we use Command.Flags from the run$Cmd function via parseApps() and Command.Run is initialized with run$Cmd.

If you know a better way to work around it, I'm not happy with this, just working around the compiler.